### DISCUSSION TITLE
Can we use 2 fonts in rofi? If yes then how?

### DISCUSSION THREAD

**Discussion by DestroyerAce** [davatorium/rofi#1768] [Q&A] [answered]
Same as the title

**Comment by DaveDavenport** [comment, marked as answer]
yes,  by setting another font for a specific widget.

**Reply by DaveDavenport** [reply]
https://github.com/davatorium/rofi/blob/next/doc/rofi-theme.5.markdown#textbox

**Reply by DestroyerAce** [reply]
Thanks mate
